### PR TITLE
Set _TRIGGER_JOB_DONE_HOOK=1 for investigation jobs

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/98862

Then passed jobs will also trigger the hooks for post investigation processing.

### CODE REVIEW

**Review by okurz — CHANGES_REQUESTED** [review]
The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

File: openqa-investigate [hunk 1]
```diff
@@ -47,7 +47,7 @@ clone() {
 
     name="$(echo "$clone_job_data" | runjq -r '.job.test'):investigate$name_suffix" || return $?
     base_prio=$(echo "$clone_job_data" | runjq -r '.job.priority') || return $?
-    clone_settings=("TEST+=:investigate$name_suffix" '_GROUP_ID=0' 'BUILD=')
+    clone_settings=("TEST+=:investigate$name_suffix" ''_TRIGGER_JOB_DONE_HOOK=1' _GROUP_ID=0' 'BUILD=')
```
Comment (Martchus) [inline openqa-investigate:50]: The quoting looks a little odd.

**Review by Martchus — CHANGES_REQUESTED** [review]

File: openqa-investigate (hunk 1 above)
Comment (perlpunk) [reply openqa-investigate:50]: Oops. fixed

Comment (perlpunk) [reply openqa-investigate]: and pushed...

**Review by Martchus — APPROVED** [review]

**Review by kalikiana — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**